README.md

```diff
@@ -1,75 +1,84 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Doctors Appointment Front-end
```

Comment:
- On a new "Acknowledgments" section of your readme, kindly give credit to Murat Korkmaz for the design.